install/kubernetes/cilium/values.yaml

```
@@ -661,10 +661,14 @@ hubble:
       # -- certmanager issuer used when hubble.tls.auto.method=certmanager.
       # If not specified, a CA issuer will be created.
       certManagerIssuerRef: {}
-    # -- base64 encoded PEM values for the Hubble CA certificate and private key.
+
+    # -- Deprecated in favour of tls.ca. To be removed in 1.13.
```

nitpicking for consistency
```suggestion
    # -- Deprecated in favor of tls.ca. To be removed in 1.13.
```